views/login.erb

```diff
@@ -0,0 +1,20 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <h1>Login</h1>
+    <% if flash[:email_error]  %> 
```

Comment:
I'm getting some strange line-ender characters that VSCode is unhappy about in this file...not sure what they are or if it's only me that's getting them!